app/models/rubygem_transfer.rb

```diff
@@ -41,20 +42,34 @@ def approved_invites
     invites.includes(:user).select { |invite| invite.user.present? && invite.role.present? }
   end
 
+  def available_rubygems
+    return Rubygem.none if created_by.blank?
+    created_by.rubygems.where(organization_id: nil).order(:name)
+  end
+
+  def selected_rubygems
+    @selected_rubygems ||= Rubygem.preload({ ownerships: :user }, :organization).where(id: rubygems)
+  end
+
+  def rubygems=(value)
+    @selected_rubygems = nil
+    super
+  end
```

Comment:
🔎 Is this used? I see we removed `belongs_to :rubygem`, but don't see a `has_many :rubygems` in its place. I don't see it called anywhere here, but haven't looked outside the changes.

Reply:
`rubygems` is an integer array on the `RubygemTransfer` model. The purpose of this method is to clear the cached `selected_rubygems` method.

It's more primarily there for the tests than the application. We could remove the cached instance variable in `selected_rubygems`, but I felt we can get away with this. What are thoughts?